modules/nf-core/cnvkit/genemetrics/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: cnvkit_genemetrics
+description: Copy number variant detection from high-throughput sequencing data
+keywords:
+  - cnvkit
+  - genemetrics
+tools:
+  - cnvkit:
+      description: |
+        CNVkit is a Python library and command-line software toolkit to infer and visualize copy number from high-throughput DNA sequencing data. It is designed for use with hybrid capture, including both whole-exome and custom target panels, and short-read sequencing platforms such as Illumina and Ion Torrent.
+      homepage: https://cnvkit.readthedocs.io/en/stable/index.html
+      documentation: https://cnvkit.readthedocs.io/en/stable/index.html
+      licence: ["Apache-2.0"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - cnr:
+      type: file
+      description: CNR file
+      pattern: "*.cnr"
+  - cns:
+      type: file
+      description: CNS file
+      pattern: "*.cns"
```

Comment:
Given the code you wrote I'd say that this file is optional.
Can you confirm, and if so add this information here?

Reply:
you are completely right. Will add that information 😉